bpfd-operator/config/bpfd-deployment/daemonset.yaml

```diff
@@ -53,23 +53,30 @@ spec:
           mountPath: /etc/bpfd/bpfd.toml
           subPath: bpfd.toml
           readOnly: true
+        # Mounted so bpfd doesn't try and create the client code
+        - name: bpfd-agent-cert
+          mountPath: /etc/bpfd/certs/bpfd-client
+          readOnly: true
           # These certs are missing in the scratch image to keep it small.  Mount
           # and use the host's instead.
         - name: host-bytecode
           mountPath: /run/bpfd/bytecode
       - name: bpfd-agent 
-        image: quay.io/bpfd/bpfd-agent:main
+        command:
+        - /bpfd-agent      
+        image: bpfd-agent-image:latest
```

Comment:
Why is it `bpfd-agent-image:latest` but `bpfd-image:main`? Seems the default tags should be the same.

Reply:
Good catch!! This is actually just a tag used by `kustomize` to configure things.... Using a command like 

```
	cd config/bpfd-operator-deployment && $(KUSTOMIZE) edit set image bpfd-operator-image=${BPFD_OPERATOR_IMG} 
	cd config/bpfd-deployment && $(KUSTOMIZE) edit set image bpfd-image=${BPFD_IMG} && $(KUSTOMIZE) edit set image bpfd-agent-image=${BPFD_AGENT_IMG}
``` 

Is actually what gets these deployments get their default images (as specified in the Makefile)